src/openklant/components/klantinteracties/models.py

```diff
@@ -0,0 +1,338 @@
+        ),
+        max_length=80,
+        blank=True,
+    )
+    correspondentieadres_adresregel3 = models.CharField(
+        _("Adresregel 3"),
+        help_text=_(
+            "Derde van het adres dat niet voorkomt in de Basisregistratie Adressen en Gebouwen."
+        ),
+        max_length=80,
+        blank=True,
+    )
+    correspondentieadres_land = models.IntegerField(
+        _("Land"),
+        help_text=_(
+            "Een code, opgenomen in Tabel 34, Landentabel, die het land (buiten Nederland) "
+            "aangeeft alwaar de ingeschrevene verblijft."
+        ),
+        validators=[MaxValueValidator(9999)],
+        blank=True,
+    )
+
+    # Bezoekadres model fields:
+    bezoekadres_nummeraanduiding_id = models.UUIDField(
+        default=uuid.uuid4,
+        unique=True,
+        max_length=40,
+        help_text=_(
+            "Identificatie van het adres bij de Basisregistratie Adressen en Gebouwen."
+        ),
+    )
+    bezoekadres_adresregel1 = models.CharField(
+        _("Adresregel 1"),
+        help_text=_(
+            "Eerste deel van het adres dat niet voorkomt in de Basisregistratie Adressen en Gebouwen."
+        ),
+        max_length=80,
+        blank=True,
+    )
+    bezoekadres_adresregel2 = models.CharField(
+        _("Adresregel 2"),
+        help_text=_(
+            "Tweede deel van het adres dat niet voorkomt in de Basisregistratie Adressen en Gebouwen."
+        ),
+        max_length=80,
+        blank=True,
+    )
+    bezoekadres_adresregel3 = models.CharField(
+        _("Adresregel 3"),
+        help_text=_(
+            "Derde deel van het adres dat niet voorkomt in de Basisregistratie Adressen en Gebouwen."
+        ),
+        max_length=80,
+        blank=True,
+    )
+    bezoekadres_land = models.IntegerField(
+        _("Land"),
+        help_text=_(
+            "Een code, opgenomen in Tabel 34, Landentabel, die het land (buiten Nederland) "
+            "aangeeft alwaar de ingeschrevene verblijft."
+        ),
+        validators=[MaxValueValidator(9999)],
+        blank=True,
+    )
+
+    # Group types:
+    correspondentieadres = GegevensGroepType(
+        {
+            "nummeraanduiding id": correspondentieadres_nummeraanduiding_id,
+            "adresregel 1": correspondentieadres_adresregel1,
+            "adresregel 2": correspondentieadres_adresregel2,
+            "adresregel 3": correspondentieadres_adresregel3,
+            "land": correspondentieadres_land,
+        },
+    )
+    bezoekadres = GegevensGroepType(
+        {
+            "nummeraanduiding id": bezoekadres_nummeraanduiding_id,
+            "adresregel 1": bezoekadres_adresregel1,
+            "adresregel 2": bezoekadres_adresregel2,
+            "adresregel 3": bezoekadres_adresregel3,
+            "land": bezoekadres_land,
+        }
+    )
+
+    class Meta:
+        verbose_name = "partij"
+        verbose_name_plural = "partijen"
```

Comment:
Since all the other strings are marked as translatable, these should too?

Reply:
I didn't know for sure if we wanted to do that, because for the already existing models they didn't use translatable strings.
Two examples are the `Klant` and `NatuurlijkPersoon` models (openklant/components/klanten/models.klanten.py)